transcode.py

```diff
@@ -6,6 +6,13 @@
 import os
 import sys
 from xml.dom import minidom
+import boto3
+from os.path import join, dirname
+from dotenv import load_dotenv
```

Comment:
No need to use dotenv.  just use the `os.environ`  let's just stick to the system libraries